Pollo/Views/Cards/PollsDateCell.swift

```diff
@@ -14,14 +14,17 @@ class PollsDateCell: UICollectionViewCell {
     var dateLabel: UILabel!
     var greyView: UIView!
     var rightArrowButtonImageView: UIImageView!
+    var indicatorDot: UIView!
```

Comment:
we should make an issue to convert views across the codebase to be `private let` where possible and then hopefully tackle it when someone has downtime